src/solver/types.jl

```diff
@@ -459,6 +460,14 @@ function AtmosConfig(;
     else
         @info "Running ClimaCore in unthreaded mode."
     end
+    if config["log_params"]
+        default_output = haskey(ENV, "CI") ? job_id : joinpath("output", job_id)
+        out_dir = parsed_args["output_dir"]
+        output_dir = isnothing(out_dir) ? default_output : out_dir
+        mkpath(output_dir)
+        logfilepath = joinpath(output_dir, "$(parsed_args["job_id"])_$FT.toml")
+        CP.log_parameter_information(toml_dict, logfilepath)
+    end
```

Comment:
You could move this function to `get_integrator`, where the `output_dir` is already defined (after `get_simulation`)

Reply:
TY!